metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -345,21 +347,35 @@
    (j/with-db-transaction
      [conn db]
      (let [table (tables/get-table-name provider concept-type)]
-       (db-result->concept-map concept-type conn (:provider-id provider)
-                               (su/find-one conn (select '[*]
-                                                         (from table)
-                                                         (where `(= :concept-id ~concept-id))
-                                                         (order-by (desc :revision-id))))))))
-  ([db concept-type provider concept-id revision-id]
-   (if revision-id
-     (let [table (tables/get-table-name provider concept-type)]
-       (j/with-db-transaction
-         [conn db]
+       (when (or
+              (= "efs-on" efs-config/efs-toggle)
+              (= "efs-only" efs-config/efs-toggle))
+         (efs/get-concept provider concept-type concept-id))
+       (when (or
```

Comment:
Can you just check to see if (not (= efs-only efs-config/efs-toggle)).

Reply:
I suppose I could do that for both. check for (not (= efs-only...)) and (not (= efs-off...))
Functionally would make no real difference, would just mean the third toggle value could be any nonsense value